docs/paying-for-lit/capacity-credit-intro.md

```diff
@@ -0,0 +1,5 @@
+# Capacity Credits
+
+
+
+Capacity Credits are an extension of ERC-721 NFTs that reserve a specific amount of requests for you to use over a period of time. The [contract](https://github.com/LIT-Protocol/lit-assets/blob/develop/blockchain/contracts/contracts/lit-node/RateLimitNFT/RateLimitNFTFacet.sol) is deployed on the Chronicle Yellowstone rollup, and can be interacted with via the Lit Relayer, or the Chronicle Yellowstone RPC endpoint: `https://yellowstone-rpc.litprotocol.com/`.
```

Comment:
Similar to what Alex said in his comment, I was thinking it would be helpful if we could aggregate exactly which requests will require Capacity Credits. I guess this could go in the delegating-credit.md file. Just thought it was a good idea.

Reply:
agree with this comment. I also think we could be a little more general here and abstract away some of the lower level details. Maybe soemthing like: "capacity credits are how you pay for usage on the Lit network. They allow you to reserve a specific amount of capacity (requests / sec) on the network over a pre-defined time interval (i.e. 1 week). In order to sign or decrypt on Lit, you'll need to hold or be delegated a credit. You can create them by minting them on the [lit rollup] or using the Lit [relayer].."